Common/Interfaces/IDataManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using System.Collections.Generic;
+using QuantConnect.Data;
+
+namespace QuantConnect.Interfaces
+{
+    /// <summary>
+    /// DataManager interface will manage the subscriptions for both the DataFeeds and the SubscriptionManager
+    /// </summary>
+    public interface IDataManager
+    {
+        /// <summary>
+        /// Gets all the current data config subscriptions that are being processed
```

Comment:
This is a little misleading -- really it's all subscriptions added to the subscription manager -- comment of a similar color for the docs below as well.

Reply:
I've updated the docs to be more explicit 👍. Also we will now have an interface for each consumer.